apps/production/views.py

```
@@ -1208,18 +1216,41 @@ def picking_list_edit(request, document_id):
-                
+
+                # Surovina už ve výdejce jídla je: PickingList má
+                # unique_together(production_order, ingredient), create() by
+                # spadl na IntegrityError 500. Uživatel má upravit existující řádek.
+                if PickingList.objects.filter(
+                    production_order=order_obj, ingredient=ingredient_obj
+                ).exists():
+                    messages.warning(
+                        request,
+                        f'Surovina {ingredient_obj.name} už je ve výdejce jídla '
+                        f'{order_obj.recipe.name} – upravte množství u existující položky.'
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)